src/scripts/main.js

```diff
@@ -1,7 +1,27 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const messageBlock = document.querySelector('div');
```

Comment:
Use `createElement` instead of `querySelector` to create new DOM elements.
<img width="1273" alt="Screenshot 2023-05-15 at 16 55 01" src="https://github.com/mate-academy/js_notification_DOM/assets/53228259/4372795c-d1f5-4095-ad0b-1cfd638eb5e8">